test/opcodes/cpy_test.c

```diff
@@ -0,0 +1,354 @@
+#include "unity.h"
+#include "cpu.h"
+
+struct BUS bus;
+struct CPU cpu;
+
+char CPY_Immediate = 0xC0;
```

Comment:
Loved that you declared the opcode as variables, better than my comments

Reply:
Thanks <3